day_7/10_speckled_frogs.js

```diff
@@ -0,0 +1,7 @@
+for (let i = 10; i > 2; i--){
+  console.log(i + " speckled frogs sat on a log \neating some most delicious bugs.\nOne jumped in the pool where its nice and cool,\nthen there were " + (i - 1) + " speckled frogs.\n")
+}if(i = 2){
```

Comment:
note that you're using assignment instead of comparison here. How would you change this to use comparison?